ScorchApp/src/main.js

```diff
@@ -16,12 +17,12 @@ import 'styles/styles.scss'
 import VueSocketio from 'vue-socket.io';
 
 Vue.use(VueResource);
+Vue.use(Notify);
 Vue.config.productionTip = false
 Vue.url.options.root = 'https://dnd-api.imightybigman.com/api';
```

Comment:
that is prod LOLOL

Reply:
that should be.... Vue.config.APIURL or somethingl iek that. Look into the config folder

Reply:
Vue.url.options.root = process.env.API_HOST;